services/webnn/ort/context_impl_ort.cc

```diff
@@ -117,7 +117,7 @@ ContextProperties ContextImplOrt::GetContextProperties() {
        /*lstm_input=*/{},
        /*lstm_cell_input=*/{},
        /*matmul_input=*/DataTypeConstraint::kFloat16To32Ints32To64,
-       /*pad_input=*/{},
+       /*pad_input=*/DataTypeConstraint::kFloat16To32,
```

Comment:
It seems ONNX Pad can support int4. We can leave a TODO and test it later.

https://onnx.ai/onnx/operators/onnx__Pad.html

Reply:
Done. Please take another look. Thanks!